Packs/Celonis/Integrations/CelonisEventCollector/README.md

```diff
@@ -0,0 +1,67 @@
+Celonis Event Collector is an integration that supports fetching audit log events.
+This integration was integrated and tested with version 4.0 of Celonis.
+
+## Configure Celonis in Cortex
+
+
+| **Parameter** | **Description**                                                                                                                    | **Required** |
+| --- |------------------------------------------------------------------------------------------------------------------------------------| --- |
+| Server URL | The endpoint URL is constructed using the team name and realm in the format: https://&lt;teamname&gt;.&lt;realm&gt;.celonis.cloud. | True |
+| Server URL | The endpoint URL is constructed using the team name and realm in the format: https://&lt;teamname&gt;.&lt;realm&gt;.celonis.cloud. | True |
+| Client ID | The Client ID to use for connection.                                                                                               | True |
+| Client Secret | The Client Secret to use for connection.                                                                                           | True |
+| Trust any certificate (not secure) |                                                                                                                                    | False |
+| Use system proxy settings |                                                                                                                                    | False |
+| Maximum number of events per fetch | Defines the maximum number of audits events per fetch cycle. Default value: 600.                                                   | True |
+
+API keys, passed in an HTTP header like this: Authorization: Bearer API_KEY.
+
+#### How to create an OAuth client and generate client ID and Client Secret:
+https://developer.celonis.com/celonis-apis/audit-log-api/#creating-an-application-and-granting-it-api-permissions
```

Comment:
Add this link below the explanation with the text of:
For the following information visit Celonis API(link the URL here)